cmd/ocagent/main.go

```diff
@@ -87,7 +89,10 @@ func runOCAgent() {
 		log.Fatalf("Could not instantiate logger: %v", err)
 	}
 
-	traceExporters, metricsExporters, closeFns, err := config.ExportersFromYAMLConfig(logger, yamlBlob)
+	// TODO(skaris): move the rest of the configs to use viper
+	v := viper.New()
```

Comment:
We forgot to set below `v.SetConfigType("yaml")` which directly goes to my suggestion to extract such code into a single helper.